helpers/auth/authentication.py

```diff
@@ -63,7 +63,34 @@ def decode_token(self):
                 'Invalid token. Please Provide a valid token!'
             }), 401
 
-    def save_user(self):
+    def andela_api_call(self, email, *expected_args):
```

Comment:
@vic3king please add a docstring to give an overview of how this function works. Ensure to highlight the parameters it takes in and what it returns.

Reply:
will do